.github/workflows/ci.yaml

```diff
@@ -50,6 +50,7 @@ jobs:
           mix local.rebar --force
           mix local.hex --force
           mix deps.get
+          mix deps.unlock --check-unused
```

Comment:
this should be a step of "Static Code Analysis"

Reply:
Moved to a step under Static Analysis, after `Retrieve Elixir Cached Dependencies`.